config/routes.rb

```diff
@@ -177,6 +177,7 @@
       resource :email_addresses, only: %i[create update]
       resource :telephones, only: %i[create update]
       get 'status/:transaction_id', to: 'transactions#status'
+      get 'status/', to: 'transactions#statuses'
```

Comment:
Does this also work for the non-`/`'d version? I believe the convention with the rest of `vets-api` is to not have a trailing slash if there is no parameter afterwards.